MetaCell/cloud-harness
# Feature - Move from alpine to debian image

Closes #CH-95

Implemented solution: 

- replaced the alpine with the debian base image in cloudharness-base image
- replace the occurrences of cloudharness-base-debian with cloudharness-base and apk to apt where found
- created a configuration script for the migration, if the keyword is found in a file then it iterates the old words and ask the user if they want to replace them with the new, a python script runs the migration as well.
- integrated the above with harness_deployment and 2 new flags (one for migration, one for migration without input from the user)

How to test this PR: 

- manually create an application in the applications folder and make this use the CLOUDHARNESS_BASE_DEBIAN image to create the application container, and also apk to install some packages
- run the harness_deployment tool with the migration flag
- check the files wiht the keyword created at the first step

- automated testing has been implemented as well to test the utils created and the perform_migration function.

### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):

### Nice to have (if relevant):
